harvester/tests/integration/test_civic_location_normalizer.py

```diff
@@ -21,3 +25,14 @@ def test_civic():
     for f in fa['features']:
         print f['name'], f['provenance']
     assert len(fa['features']) == 1
+
+
+def test_civic2():
+    evidence = json.loads(CIVIC2)
+    gene_data = {'gene': 'KRAS', 'civic': {'variants': [evidence]}}
```

Comment:
Why `'gene': 'KRAS'` here? Is this to test if the gene name from the evidence is properly propagated? If so, we should add an assertion for that. Also, this belonged in the #99 review--sorry!

Reply:
thanks, good catch